app/assets/javascripts/visualizations/highvis/regression.coffee

```diff
@@ -364,12 +319,78 @@ $ ->
       mean
 
     # Normalize
-    normalizeData = (points, mean, sigma) ->
-      (point - mean) / sigma for point in points
+    normalizeData = (points, type) ->
+      max = Math.max.apply(null, points)
+      min = Math.min.apply(null, points)
+      ret = if type == globals.REGRESSION.LOGARITHMIC
```

Comment:
would you mind bringing the if down the next line. it will end up looking like this
```
ret =
  if type == globals.REGRESSION.LOGARITHMIC then points
  else points.map((y) -> ((y - min) / (max - min)) + 1)
```